client/src/components/NavBar/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import HomeIcon from '@material-ui/icons/Home';
+import ControlPointIcon from '@material-ui/icons/ControlPoint';
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
+import Grid from '@material-ui/core/Grid';
+import { Link } from 'react-router-dom';
+import './style.css';
+
+const useStyles = makeStyles((theme) => ({
+  icons: {
+    width: 50,
+    height: 40,
+    color: theme.customColors.white,
+  },
+  container: {
+    backgroundColor: 'green',
+  },
+  item: {
+    width: '25%',
+  },
+  appBar: {
+    bottom: 0,
+    top: '88%',
+  },
+}));
+
+function NavBar() {
+  const classes = useStyles();
+
+  return (
+    <>
+      <AppBar
+        position="fixed"
+        style={{
+          alignItems: 'center',
+        }}
+        className={classes.appBar}
+      >
+        <Toolbar
+          style={{
+            justifyContent: 'center',
+            alignItems: 'center',
+            width: '80%',
```

Comment:
These could also be moved into a named class inside the makeStyles at the top of the file, for sake of consistency.